src/tcg-to-deckbox.py

```diff
@@ -61,45 +167,88 @@ def getPathPrefix():
-        row["Name"]=row["Name"].replace(" (Borderless)","")
-        #For BFZ lands...there's no differentiator from the full arts and the non full arts.
-        row["Name"]=row["Name"].replace(" - Full Art","")
+        ####################################################################
+        # Map Specific Card Conditions
+        ####################################################################
 
-        #Very specifc conditons
+        # For BFZ lands...there's no differentiator from the full arts and the non full arts.
+        row["Name"] = row["Name"].replace(" - Full Art", "")
+
+        # War of the Spark Alternate Arts handled differently
         if "(JP Alternate Art)" in row["Name"] and row["Edition"] == "War of the Spark":
             row["Edition"] = "War of the Spark Japanese Alternate Art"
-            row["Name"]=row["Name"].replace(" (JP Alternate Art)","")
+            row["Name"] = row["Name"].replace(" (JP Alternate Art)", "")
+
+        # Buy a Box Promos worked a little differently with Ixalan
+        if row["Name"] in ixalan_bab and row["Edition"] == "Buy-A-Box Promos":
+            row["Edition"] = "Black Friday Treasure Chest Promos"
+            skip_scryfall_names = True
 
+        # Handle Mystery Booster Test Cards, the 2021 release differentiates by Edition
+        # on deckbox, while tcgplayer differentiates by name appending '(No PW Symbol)'
+        if (
+            "(No PW Symbol)" in row["Name"]
+            and row["Edition"] == "Mystery Booster: Convention Edition Exclusives"
+        ):
+            row["Edition"] = "Mystery Booster Playtest Cards 2021"
 
-        # Remove numbers, mostly for lands, but for some other special cases (M21 Teferi)
-        row["Name"] = re.sub(r" \(\d+\)", "", row["Name"])
+        ####################################################################
+        # Handle General Card Conversions
+        ####################################################################
+
+        # Remove all Parentheses at the end of cards
+        row["Name"] = re.sub(r" \(.*\)", "", row["Name"])
         replace_strings(row, "NAMES", "Name")
+        # We need to do a little extra work on dual faced cards, because deckbox is inconsistent with whether it refers to cards by both names or just the front face.
+        if row["Name"] in scryfall_data:
+            deckbox_request_url = DECKBOX_URL + urllib.parse.quote(
+                scryfall_data[row["Name"]]
+            )
+            with requests.get(deckbox_request_url) as deckbox_response:
+
+                # If we are not redirected to a new page, then we should only use the front face name
+                if deckbox_request_url == deckbox_response.url:
+                    print(
+                        "Dual name for '%s' found on deckbox, the dual name will be used for the import."
+                        % (scryfall_data[row["Name"]], scryfall_data[row["Name"]])
```

Comment:
Format string has a single specifier but 2 arguments.  Removing the second argument cleared it up.  Only issue that I hit when uploading another set of cards this evening.